src/Core/src/Platform/Android/Navigation/NavigationViewFragment.cs

```diff
@@ -14,10 +15,24 @@ public class NavigationViewFragment : Fragment
 		NavigationLayout? _navigationLayout;
 		FragmentContainerView? _fragmentContainerView;
 
-		NavigationLayout NavigationLayout =>
-			_navigationLayout ??= (FragmentContainerView.Parent as NavigationLayout)
-			?? (FragmentContainerView.Parent?.Parent as NavigationLayout)
-			?? throw new InvalidOperationException($"NavigationLayout cannot be null here");
+		// TODO FIX ME
```

Comment:
Did this get pulled over from old code, or is it a new TODO? If it's a new one, can we get some more context/info on what needs to be fixed?

Reply:
👍